### DISCUSSION TITLE
12월 9일 미팅은 온라인으로 진행하는 건 어떨까요?

### DISCUSSION THREAD

**Discussion by jongfeel** [silqwer/book-club#11] [General]
오프라인으로 홍대에서 보는 게 좋기는 한데
이번주 금요일은 제가 회사를 안나올 거라...

23일은 홍대에서 보는 걸로 해보시죠
11시 쯤 만나서 얘기하고 점심도 먹고
물론 부담스러우시면 책 얘기만 해도 됩니다 ㅎㅎ

**Comment by silqwer** [comment]
넵넵 이번주 금요일 11시 홍대에서 봬욥
저는 여기서 기다리고 있을께욥 

 [네이버 지도]
스타벅스 동교점
서울 마포구 월드컵북로 22
https://naver.me/x0hpIoRG

**Reply by jongfeel** [reply]
다시 협의해서 12월 9일도 홍대에서 보는 걸로 결정!
